sdk/tests/conformance2/renderbuffers/multisampled-renderbuffer-initialization.html

```diff
@@ -52,8 +52,10 @@
 
     runTest(gl, gl.canvas.width, gl.canvas.height, 0);
```

Comment:
Can you define some const variable (or whatever in js) to be 0, 1, 2, like ALLOCATE_ATTACH = 0, ATTACH_ALLOCATE = 1, and ALLOCATE_ATTACH_ALLOCATE_ANOTHER_SIZE = 2? This will make code much easier to read.